internal/command/arguments/extended.go

```diff
@@ -119,8 +120,44 @@ func (o *Operation) Parse() tfdiags.Diagnostics {
 			continue
 		}
 
-		o.Targets = append(o.Targets, target.Subject)
+		targetables = append(targetables, target.Subject)
 	}
+	return targetables, diags
+}
+
+func parseRawTargetsAndExcludes(targets []string, excludes []string) ([]addrs.Targetable, []addrs.Targetable, tfdiags.Diagnostics) {
+	var parsedTargets []addrs.Targetable
+	var parsedExcludes []addrs.Targetable
+	var diags tfdiags.Diagnostics
+
+	if len(targets) > 0 && len(excludes) > 0 {
+		diags = diags.Append(tfdiags.Sourceless(
+			tfdiags.Error,
+			"-target and -exclude flags used together",
```

Comment:
Suggestion: `"Invalid combination of arguments"`

Reply:
Done in 90554e23b7f1e4f193e86a6a454f895925587aca